### DISCUSSION TITLE
Gratitude

### DISCUSSION THREAD

**Discussion by jspiro** [fabrizioschiavi/pragmatapro#212] [General]
Fabrizio,

I've been following the barrage of issues that came in the last week, and I'm so impressed. No matter when we bought Pragmata Pro, you continue to consistently provide huge updates to it and ask nothing more from us for it. It's clearly a labor of love for you and a passion for your users.

Thank you for the mere existence of this repo and for being the amazing artist that reads and addresses all of our wishes and nitpicks to produce the world's very best monospaced font, with uses far beyond terminals and editors.

<3
- @jspiro, and probably most (if not all) of this community

**Comment by fabrizioschiavi** [comment]
I am touched. Thank you @jspiro for this unexpected message!

Thanks to all of you because without your contribution PragmataPro should have been a project without energy.
We design PragmataPro, **together**
